cart.js

```diff
@@ -1,20 +1,12 @@
 const navCart = document.getElementById("nav__cart");
 const cartMenu = document.getElementById("nav__cart-menu");
-// const navHamburger = document.getElementById("nav__hamburger");
-// const hamburgerMenu = document.getElementById("nav__hamburger-menu");
-// const hamburgerIcon = document.getElementById("nav__hamburger-hamburgerIcon");
-// const closeIcon = document.getElementById("nav__hamburger-closeIcon");
 
 navCart.addEventListener("click", toggleCart);
 
 function toggleCart() {
-  if(hamburgerMenu.classList.contains("nav__hamburger-menu--on")) {
-    hamburgerMenu.classList.remove("nav__hamburger-menu--on");
-    hamburgerMenu.classList.add("nav__hamburger-menu--off");
-    closeIcon.classList.remove("nav__hamburger-menu--on");
-    closeIcon.classList.add("nav__hamburger-menu--off");
-    hamburgerIcon.classList.remove("nav__hamburger-menu--off");
-    hamburgerIcon.classList.add("nav__hamburger-menu--on");
-}
-cartMenu.classList.toggle("nav__cart-menu--on");
+  if(!hamburgerMenu.classList.contains("--off")) {
+    hamburgerMenu.classList.toggle("--off");
```

Comment:
Previous class name was nice `nav__hamburger-menu--off`. Also if you wish you can call it `hamburger-menu--off`

Reply:
--Off is a state of a component not class name, that is used for multiple elements. You can rename is as "hidden" or something like that